### PR TITLE
Revert "chore: Don't include `new` in responses (#34617)"

### PR DESCRIPTION
**Pull request by sharat87**
**/test ImportExport git**

> [!CAUTION]
> 🔴 🔴 🔴 Some tests have failed.
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9761964432>
> Commit: 44fd1e67988bee13cbb8a3353825f04304daaf93
> <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9761964432&attempt=2&selectiontype=test&testsstatus=failed&specsstatus=fail" target="_blank">Cypress dashboard</a>.
> Tags: @tag.ImportExport, @tag.Git
> The following are new failures, please fix them before merging the PR: <ol>
> <li>cypress/e2e/Regression/ClientSide/Git/GitSync/GitSyncedApps_spec.js</ol>
> <a href="https://internal.appsmith.com/app/cypress-dashboard/identified-flaky-tests-65890b3c81d7400d08fa9ee3?branch=master" target="_blank">List of identified flaky tests</a>.

### CODE REVIEW

File: app/server/appsmith-interfaces/src/main/java/com/appsmith/external/models/BaseDomain.java [hunk 1]
```diff
@@ -79,7 +78,7 @@ public abstract class BaseDomain implements Persistable<String>, AppsmithDomain,
     protected Set<Policy> policies = new HashSet<>();
 
     @Override
-    @JsonIgnore
+    @JsonView(Views.Public.class)
```
Comment (sondermanish) [inline app/server/appsmith-interfaces/src/main/java/com/appsmith/external/models/BaseDomain.java:81]: @sharat87 Now we are expecting this to be part of serialisation as well? i.e. git flows?

Comment (sondermanish) [reply app/server/appsmith-interfaces/src/main/java/com/appsmith/external/models/BaseDomain.java:81]: If yes then let's add the git annotation as well which works with the cleanup whitelisting

Comment (sharat87) [reply app/server/appsmith-interfaces/src/main/java/com/appsmith/external/models/BaseDomain.java:81]: This is a revert PR, reverting #34617. Git was not there before, so this revert isn't adding it either. Do we need it? 🤔

Comment (sondermanish) [reply app/server/appsmith-interfaces/src/main/java/com/appsmith/external/models/BaseDomain.java:81]: Nope, AFAIR we don't require this for our ops. we should be good.

**Review by sondermanish — APPROVED** [review]

**Pull request merged**